massgov/mayflower
# patternlab/DP-15872: Update styling of Campaign Feature Organism

### Minor
#### Changed
- (Patternlab) [CampaignFullWidthFeature] DP-15872: Replaces campaign feature full width molecule with card molecule. #742
- (Patternlab) [CampaignFeature2up] DP-15872: Updates campaign feature 2up to consume new card molecule in secondary variant and updates molecule scss. #742
- (Patternlab) [CampaignFeature] DP-15872: Updates campaign feature organism to consume primary card molecule and adds organism scss. #742
#### Added
- (Patternlab) [Card] DP-15872: Adds card molecule with primary (default) and secondary usage variant. #742

### Steps to Test
1. `git pull patternlab/DP-15872--campaign-features-v0.2`
2. start local server (`cd patternlab/styleguide` &`nvm use & npm i` & `gulp`)
3. Visit local server in browser go to http://localhost:3000/?p=organisms-campaign-feature and confirm matches mocks here https://eotss.invisionapp.com/d/main#/projects/prototypes/18386513
